### PR TITLE
Polish readme

### PR DESCRIPTION
**Pull request by hotchemi**
The reason I do is that make README more neat and make it approachable for new-comer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -254,19 +225,6 @@ repositories {
 }
 ```
 
-And on your **app module** `build.gradle`:
```
Comment (hotchemi) [inline README.md]: I deleted android-apt because I saw some user was confused about which to use.

File: README.md [hunk 2]
```diff
@@ -4,18 +4,20 @@
 
 ![image](https://raw.githubusercontent.com/hotchemi/PermissionsDispatcher/master/art/logo.png)
 
-PermissionsDispatcher provides a simple annotation-based API to handle runtime permissions in Android Marshmallow.
-
-[Runtime permissions](https://developer.android.com/preview/features/runtime-permissions.html) are great for users, but can be tedious to implement correctly for developers, requiring a lot of boilerplate code.
+PermissionsDispatcher provides a simple annotation-based API to handle runtime permissions in Android Marshmallow, **with 100% reflection-free.**.
```
Comment (aurae) [inline README.md]: Let's remove the *with* at the end:

> PermissionsDispatcher provides a simple annotation-based API to handle runtime permissions in Android Marshmallow, **~with~ 100% reflection-free.**

File: README.md [hunk 3]
```diff
@@ -4,18 +4,20 @@
...
 
-**The library is 100% reflection-free.**
-
 ## Usage
 
 Here's a minimum example, in which we register a `MainActivity` which requires `Manifest.permission.CAMERA`.
 
+### 0. Prepare AndroidManifest
+
+Add following line to `AndroidManifest.xml`
```
Comment (aurae) [inline README.md]: Let's add an article here, as well as a colon at the end.

> Add **the** following line to `AndroidManifest.xml`:

File: README.md [hunk 4]
```diff
@@ -4,18 +4,20 @@
...
+ 
+`<uses-permission android:name="android.permission.CAMERAW" />`
```
Comment (aurae) [inline README.md]: Typo, should say `android.permission.CAMERA`

File: README.md (hunk 1 above)
Comment (aurae) [inline README.md]: Good choice, absolutely!

**Comment by hotchemi** [comment]
@aurae Thx! addressed 2f1a24e033e7014ddeb1cf37f7b89c986a199b37.

**Pull request merged**